build.gradle

```
@@ -31,6 +31,9 @@ dependencies {
     runtimeOnly 'org.postgresql:postgresql'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'org.springframework.security:spring-security-test'
+
+    // https://mvnrepository.com/artifact/org.json/json
+    implementation("org.json:json:20230618")
```

Try and remain consistent with the implementation calls in the rest of the file. Mvn Central calls this the "Gradle (short)" version

```suggestion
    implementation 'org.json:json:20230618'
```